jsk_fetch_robot/jsk_fetch_startup/config/install_supervisor.sh

```diff
@@ -8,19 +8,14 @@ current_prefix_path="${prefix_paths[0]}"
 set -x
 
 cd $jsk_fetch_startup/supervisor_scripts
-for file in $(ls ./*.conf); do
-    sudo cp $file /etc/supervisor/conf.d/
-    sudo chown root:root /etc/supervisor/conf.d/$file
-    sudo chmod 644 /etc/supervisor/conf.d/$file
-    echo "copied $file to /etc/supervisor/conf.d"
+for file in $(ls *.conf); do
+    sudo ln -s $jsk_fetch_startup/supervisor_scripts/$file /etc/supervisor/conf.d/$file
 done
 
```

Comment:
```suggestion
sudo supervisorctl reread
```

今回のPRと直接関係ないのですが、 supervisord は conf を置き換えただけでは設定が反映されないので、installスクリプトで読み込み直しまで行いたいです。`update` (各プロセスがrestartする) と `reread` (実行中のプロセスはrestartしない) のどちらがいいかは議論の余地がありそうですが、実行直後の挙動は `reread` だと現在と変わらないため `reread` のほうがいいかなと思っています。実際に実行されているプロセスへ反映させるには `restart` が必要になります。